bigquery/scheduled_queries/append-monthly-users-from-cloudfront-logs.sql

```diff
@@ -1,96 +1,155 @@
   #Recalculates the table of monthly users taken from Cloudfront logs for each monthly user within this month,
   #and appends that to monthly users from previous months (which won't change over the course of the month).
   #Should overwrite the CALCULATED_monthly_users_from_cloudfront_logs table daily to accomplish this.
-  (
+WITH
+  cloudfront_logs AS (
   SELECT
-    month,
-    type,
-    device_category,
-    search_parameters,
-    vacancies_viewed_slugs,
-    vacancies_with_gmi_clicks_ids,
-    utm_sources,
-    utm_campaigns,
-    utm_mediums,
-    job_alert_destination_links,
-    job_alert_destinations,
-    referrers,
-    vacancies_viewed_slugs IS NOT NULL AS viewed_a_vacancy,
-    vacancies_with_gmi_clicks_ids IS NOT NULL AS clicked_get_more_information,
-    "subscription" IN UNNEST(utm_sources) AS from_job_alert,
-    ARRAY_LENGTH(search_parameters) AS unique_searches,
-    ARRAY_LENGTH(vacancies_viewed_slugs) AS vacancies_viewed,
-    ARRAY_LENGTH(vacancies_with_gmi_clicks_ids) AS vacancies_with_gmi_clicks
+    DATE_TRUNC(date, MONTH) AS month,
+    DATETIME(date,
+      time) AS datetime,
+    c_ip,
+    cs_uri_stem,
+    cs_referer,
+    c_user_agent,
+    cs_uri_query,
+    sc_content_type
```

Comment:
`c`, `sc`, and `cs` are opaque to a newcomer

Reply:
Though perhaps they are not used enough to be worth renaming as I see you are just copying them from the table column names.

Reply:
Yeah, it's how the server log is structured. But agreed, there is meaning there that isn't clear, so I'll put a comment in at least.